openmc/mesh.py

```diff
@@ -2096,6 +2122,15 @@ def dimension(self):
     def n_dimension(self):
         return 3
 
+    @property
+    @statepointcheck
+    def indices(self):
+        return [(i,) for i in range(self.n_elements)]
```

Comment:
```suggestion
        return [i for i in range(self.n_elements)]
```
Is it worth making a simpler list instead of a list of single value tuples

Reply:
The spec for this property is a list of tuples, so this would likely break downstream use. Also see comment above on 3-tuples for structured meshes.